src/main/java/pages/UpcomingPage.java

```diff
@@ -0,0 +1,20 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class UpcomingPage {
```

Comment:
UpcomingPage is not clear I think you could call it LeftPanelPage or LeftPanelSecction